src/pip/_internal/download.py

```diff
@@ -46,11 +47,17 @@
 from pip._internal.utils.ui import DownloadProgressProvider
 from pip._internal.vcs import vcs
 
+try:
+    import OpenSSL  # noqa
```

Comment:
This is going to load a C library on Windows will it not? In other words, this should probably be guarded by an ``if not WINDOWS:``.

Reply:
I'd missed this comment. Pushing a fix.